From 3ba891d560a7a912f1b4216f3109c04e0ca97c79 Mon Sep 17 00:00:00 2001
From: Meng Wei <wei.meng@intel.com>
Date: Mon, 10 Dec 2018 11:09:11 +0800
Subject: [PATCH 0994/1214] media: Disable CSI EOF IRQ

Disabling CSI EOF IRQ to decrease 1 CPU wake up for every frame for
IPU4 and IPU4P

Change-Id: I4517078eed3076f1c6e987872641556c62f6bc65
Tracked-On: PKT-1732
Signed-off-by: Hongyu Yi <hongyu.yi@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/pci/intel/ipu-isys-video.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/media/pci/intel/ipu-isys-video.c b/drivers/media/pci/intel/ipu-isys-video.c
index d3815d3..d149354 100644
--- a/drivers/media/pci/intel/ipu-isys-video.c
+++ b/drivers/media/pci/intel/ipu-isys-video.c
@@ -1615,7 +1615,9 @@ int ipu_isys_video_set_streaming(struct ipu_isys_video *av,
 		if (ip->csi2) {
 			if (ip->csi2->stream_count == 1) {
 				v4l2_subdev_call(esd, video, s_stream, state);
+#if defined(CONFIG_VIDEO_INTEL_IPU4) || defined(CONFIG_VIDEO_INTEL_IPU4P)
 				ipu_isys_csi2_wait_last_eof(ip->csi2);
+#endif
 			}
 		} else {
 			v4l2_subdev_call(esd, video, s_stream, state);
-- 
2.7.4

